docs/manuals/core/config/data-node-config.md

```diff
@@ -163,7 +168,9 @@ In lines 7-10, we configure a basic CSV data node with the id "historical_temper
 default `SCENARIO`. The default path corresponds to the file `path/hist_temp.csv`. The property _has_header_ being True,
 representing the CSV file has a header.
 
-In lines 12-14, we add another CSV data node configuration with the id "sales_history". The default `SCENARIO` scope
+In lines 12-15, we configure another CSV data node with the id "log_history". It uses the default `SCENARIO` scope again. The default path is the path to the CSV file `path/hist_log.csv`. The _exposed_type_ provided will be "modin" exposed type.
+
+In lines 17-19, we add another CSV data node configuration with the id "sales_history". The default `SCENARIO` scope
 is used again. Since we have a custom class pre-defined for this CSV file (`SaleRow`), we provide it as the
```

Comment:
No fixed size for the file name either.

Reply:
I do mean the \`SaleRow\`, yes. If it's not a file, then the text is weird: (_for this CSV file (filename)_).
I guess you can rephrase:
Since we have a custom class called \`SaleRow\` that is defined for this CSV file.